stable_plugins/classical_ml/data_preparation/wu_palmer.py

```diff
@@ -417,8 +424,8 @@ def add_similarities_for_entities(
 
     for val1 in values1:
         for val2 in values2:
-            if val1 is None or val2 is None:
-                sim = None
+            if val1 is None or val1 == "" or val2 is None or val2 == "":
```

Comment:
move check for val 1 to outer loop and maybe combine None and empty checks into Falsy checks:

```python
if not val1:  # catches None and empty strings
    continue
```

Also, don't use an else after a continue or return in an if